packages/thirdweb/src/react/core/hooks/useTransactionDetails.ts

```diff
@@ -151,9 +156,11 @@ export function useTransactionDetails({
         chainMetadata.data?.nativeCurrency?.symbol || "ETH";
       const tokenSymbol = tokenInfo?.symbol || nativeTokenSymbol;
 
-      const totalCostWei = erc20Value
-        ? erc20Value.amountWei
-        : (value || 0n) + (gasCostWei || 0n);
+      const totalCostWei =
+        erc20Value &&
+          erc20Value.tokenAddress.toLowerCase() !== NATIVE_TOKEN_ADDRESS
```

Comment:
Don't think erc20 value can be native token address, but I guess it doesn't hurt to check in case ppl do it?

Reply:
In the playground example it is (how I caught this)